ui/app/AppLayouts/Wallet/views/AccountContextMenu.qml

```diff
@@ -11,32 +11,44 @@ import utils 1.0
 StatusMenu {
     id: root
 
-    property var account
+    property var account: null
+    property var getNetworkShortNames: (chainIds) => { return "" }
```

Comment:
My proposition is to take advantage and improve a bit the API of that component to make it more explicit, e.g.:

```
property string address
property string prefix
property string name
property bool hideFromTotalBalance
```

Then all properties are typed, it's not necessary to read code to infer what's the assumed structure of `account`  and it's not needed to take callback. Moreover the component is more "pure ui" as there is no logic for obtaining prefix.

It would be also nice (but not necessary ofc) to create a storybook page for that component.

`import "../stores"` can be removed.

Reply:
@Cuteivist Have you pushed your amendments?